package.json

```diff
@@ -36,7 +36,7 @@
     "localforage-observable": "2.0.1",
     "mark.js": "8.11.1",
     "marked": "github:mattermost/marked#87769262aa02e1784570f61f4f962050e07cc335",
-    "mattermost-redux": "github:mattermost/mattermost-redux#97f0744b822f1114aa952115527ce8aae0bcc422",
+    "mattermost-redux": "github:mattermost/mattermost-redux#2c5ca00792fc5785366c4877275628de4aedb24c",
```

Comment:
was this version bump intended?

Reply:
Yep, sorry should have included the related PR. Will add to description.